blackbox/docs/protocols/http.txt

```diff
@@ -86,7 +92,7 @@ The same query using question marks as placeholders looks like this::
 .. note::
 
     With some queries the row count is not ascertainable. In this cases
-    rowcount is -1.
+    rowcount is `-1`.
```

Comment:
good catch, thanks. there were a number of existing backtick syntax mistakes in the doc. fixed them all up now